server/api/router.go

```diff
@@ -374,5 +375,22 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	// Deprecated: use /pd/api/v1/ping instead.
 	rootRouter.HandleFunc("/ping", func(w http.ResponseWriter, r *http.Request) {}).Methods("GET")
 
+	rootRouter.Walk(func(route *mux.Route, router *mux.Router, ancestors []*mux.Route) error {
+		serviceLabel := route.GetName()
```

Comment:
I prefer to directly use the handler function name to avoid using the string literals.

Reply:
> I prefer to directly use the handler function name to avoid using the string literals.

Refactored it